Makefile

```diff
@@ -128,6 +171,27 @@ $(APP).imx: $(APP).bin $(APP).dcd
 	# Copy entry point from ELF file
 	dd if=$(APP) of=$(APP).imx bs=1 count=4 skip=24 seek=4 conv=notrunc
 
+$(APP)_manifest: TAMAGO_SEMVER=$(shell ${TAMAGO} version | sed 's/.*go\([0-9]\.[0-9]*\.[0-9]*\).*/\1/')
+$(APP)_manifest:
+	@if [ "${APPLET_PRIVATE_KEY}" == "" ]; then \
+		echo 'You need to set the APPLET_PRIVATE_KEY variable to a valid signing key path'; \
+		exit 1; \
+	fi
+
+	# Create manifest
+	@echo ---------- Manifest --------------
+	#go run github.com/transparency-dev/armored-witness/cmd/manifest@b0f68ecc82043fc79384f7da7c0d44ffc1811976 \
+	go run github.com/transparency-dev/armored-witness/cmd/manifest \
```

Comment:
Do we want the pinned command above?

Reply:
Ah, this needs transparency-dev/armored-witness#21 to go in first and then pinned to that version - sorry, I would've tagged this as draft but I missed that.

Reply:
I've added the pin now that the other PR is in